python/lsst/pipe/tasks/processCcd.py

```diff
@@ -127,8 +127,15 @@ class ProcessCcdTask(pipeBase.CmdLineTask):
     RunnerClass = pipeBase.ButlerInitializedTaskRunner
     _DefaultName = "processCcd"
 
-    def __init__(self, **kwargs):
-        butler = kwargs.pop('butler')
+    def __init__(self, butler=None, refObjLoader=None, **kwargs):
+        """!
+        @param[in] butler  The butler is passed to the refObjLoader constructor in case it is
+            needed.  Ignored if the refObjLoader argument provides a loader directly.
+        @param[in] refObjLoader  An instance of LoadReferenceObjectsTasks that supplies an
+            external reference catalog.  May be None if the desired loader can be constructed
+            from the butler argument or all steps requiring a reference catalog are disabled.
+        @param[in,out] kwargs  other keyword arguments for lsst.pipe.base.CmdLineTask
+        """
         pipeBase.CmdLineTask.__init__(self, **kwargs)
         self.makeSubtask("isr")
         self.makeSubtask("charImage", butler=butler)
```

Comment:
Oops.  I somehow failed to see this comment until now (that's what I get for relying on email to find GitHub comments).  Will fix.